cfme/tests/automate/test_class.py

```diff
@@ -3,28 +3,15 @@
 import pytest
 
 from cfme import test_requirements
-from cfme.automate.explorer.domain import DomainCollection
-from cfme.utils.blockers import BZ
 from cfme.utils.update import update
 
 pytestmark = [test_requirements.automate]
 
 
-@pytest.fixture(scope='module')
-def domain(appliance):
-    dc = DomainCollection(appliance)
-    d = dc.create(
-        name='test_{}'.format(fauxfactory.gen_alpha()),
-        description='desc_{}'.format(fauxfactory.gen_alpha()),
-        enabled=True)
-    yield d
-    d.delete()
-
-
 @pytest.fixture(
     scope="function",
     params=["plain", "nested_existing"])
-def namespace(request, domain):
+def get_namespace(request, domain):
```

Comment:
As you refactoring this @ganeshhubale I would like in state of `return` we can go with `yield`. It will help you to clean what you created. what you say?

Reply:
Yes, I am agree with you.
Used ```yield``` in get_namespace.